barcode_detection/decoding/decode.py

```diff
@@ -0,0 +1,8 @@
+from abc import ABC, abstractmethod
+import numpy as np
```

Comment:
Let's divide imports into several types
1. `import ...`
2. `from ... import ...`
3. `from barcode_detection... import ...`
Every group is divided by line break and sorted alphabetically